tap_freshdesk/__init__.py

```diff
@@ -165,13 +185,14 @@ def sync_tickets():
                     singer.write_record("time_entries", subrow, time_extracted=singer.utils.now())
 
         except HTTPError as e:
-            if e.response.status_code == 403:
+            # 404 is being returned for deleted tickets and spam
+            if e.response.status_code == 403 or e.response.status_code == 404:
                 logger.info("The Timesheets feature is unavailable. Skipping the time_entries stream.")
```

Comment:
Can this be changed to split the 403 and 404 responses so that the log message makes sense in the event of a 404?

Something like `"Could not retrieve timesheets for ticket id {}. This may be caused by tickets marked as spam or deleted.".format(ticket_id)`